open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,33 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service that they would
+        that they would prefer response to be paginated. If pageSize is specified, the service should return exactly the number of items specified by it.
```

Comment:
I think the part about pageSize can be removed since it's described again in the pageSize description.